app/views/dashboard/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="container">
+  <h2>Welcome <%= current_user.first_name %></h2>
+  
+  <h4>Here are your current events:</h4>
+
+  <% @presenter.events.each do |event| %>
+    <h5><%= event.name %></h5>
+    <% event.roles.each do |role| %>
+      </p>Your role is: <%= role.name %></p>
```

Comment:
@meganft If they have more than one role I wonder if doing something like this makes sense:

```ruby
</p>Your role(s): <%= roles.pluck(:name).join(", ") %></p>
```